recipe/meta.yaml

```diff
@@ -1,21 +1,20 @@
 {% set name = "r-essentials" %}
-{% set version = "3.5.1" %}
 
 package:
   name: {{ name }}
-  version: {{ version }}
+  version: {{ r_base }}
 
 build:
   noarch: generic
   # r-rbokeh is not available under Windows
-  number: 2000
+  number: 2001
 
 requirements:
     host:
         - r-base
     run:
         - r-base
-        - r-recommended {{ version }}
+        - r-recommended {{ r_base }}
```

Comment:
It would be better not to have this explicit pin at all.
like we did on the r package: https://github.com/conda-forge/r-feedstock/blob/master/recipe/meta.yaml

Reply:
Done via https://github.com/conda-forge/r-essentials-feedstock/pull/5 which includes also this PR,